docs/how_to_add_packages.md

```diff
@@ -45,7 +45,7 @@ This process helps conan-center-index against spam and malicious code. The proce
 
 The specific steps to add new packages are:
 * Fork the [conan-center-index](https://github.com/conan-io/conan-center-index) git repository, and then clone it locally.
-* Create a new folder with the Conan package recipe (`conanfile.py`) in the correct folder.
+* Copy a template from [package_templates](package_templates) folder in the correct folder, which fits your project. Read their [documentation](package_templates/README.md) to find more information.
```

Comment:
what is the `correct folder` here?

Reply:
yes, bad explained. It should a the new package folder. An example should be added to illustrate.

Reply:
Updated, now it points recipe folder as target